app/exec/build/tasks/sign.ts

```diff
@@ -0,0 +1,159 @@
+import admZip = require("adm-zip");
+import archiver = require("archiver");
+import del = require('del');
```

Comment:
Please match the code formatting style of the rest of the repo (if you grab the Prettier VSCode extension this is very easy)